Framework/Crystal/src/SCDCalibratePanels2ObjFunc.cpp

```diff
@@ -143,8 +143,8 @@ void SCDCalibratePanels2ObjFunc::function1D(double *out, const double *xValues,
     V3D hkl =
         V3D(boost::math::iround(pk.getH()), boost::math::iround(pk.getK()),
             boost::math::iround(pk.getL()));
-    if (hkl == UNSET_HKL)
-      throw std::runtime_error("Found unindexed peak in input workspace!");
+    // if (hkl == UNSET_HKL)
+    //   throw std::runtime_error("Found unindexed peak in input workspace!");
```

Comment:
Is this meant to be commented out?

Reply:
Apparently there are un-indexed peaks in the peak workspace after passing it through IndexPeaks, so I have to comment them out.

Reply:
So should this be removed? Or will this be addressed in a later PR?

Reply:
This should be addressed when we get to changing the objective function.  I would like to keep it in as comment for now as a reminder.